markdown/blog/asyncapi-bounty-program-2024.md

```diff
@@ -0,0 +1,71 @@
+
+- 32 GitHub issues were resolved, [16 of which](https://github.com/orgs/asyncapi/projects/36/views/4?filterQuery=-no%3Alevel+-round%3A%22bounty%2F2023-pilot%22&sliceBy%5BcolumnId%5D=77803059&sliceBy%5Bvalue%5D=advanced) had complexity level `Advanced`;
+
+- [50+ PRs](https://github.com/orgs/asyncapi/projects/36/views/4?sliceBy%5BcolumnId%5D=78904519&filterQuery=no%3Aresponsible+-round%3A%22bounty%2F2023-pilot%22+) were merged in total;
+
+- [8000+ USD](https://opencollective.com/asyncapi/expenses?limit=30&tag=bounty&period=2023-09-30T21%3A00%3A00.000Z%E2%86%92all&collectiveSlug=asyncapi&status=PAID) were paid out as rewards.
+
+### Benefits for FOSS Community
+- At least [one issue](https://github.com/springwolf/springwolf-core/issues/820#issuecomment-2257879523) was closed in a third-party project due to a fixed bug in AsyncAPI's software.
+
+
+### Benefits for AsyncAPI GitHub Organization
+Thanks to the Bounty Program Participants AsyncAPI got:
+
+- [Automation in Technical Steering Committee voting process](https://github.com/orgs/asyncapi/projects/36/views/4?sliceBy%5BcolumnId%5D=78904519&sliceBy%5Bvalue%5D=community%231093&filterQuery=) ([Aayush Saini](https://github.com/AayushSaini101)): New automation and the [process around voting](https://github.com/asyncapi/community/blob/master/voting.md) have been introduced, as well as [transparent summaries](https://github.com/asyncapi/community/blob/master/TSC_VOTING_OVERVIEW.md) of each voting activity.
+
+- New monorepo setup in [generator](https://github.com/orgs/asyncapi/projects/36/views/4?filterQuery=&sliceBy%5BcolumnId%5D=78904519&sliceBy%5Bvalue%5D=generator%231044) and [parser-js](https://github.com/orgs/asyncapi/projects/36/views/4?filterQuery=&sliceBy%5BcolumnId%5D=78904519&sliceBy%5Bvalue%5D=parser-js%23963) ([Ayush Nautiyal](https://github.com/ayushnau)): Consolidation of smaller repositories into these two larger projects helped unify the dispersed community within the organization, fostering collaboration in a single location and accelerating development.
+
+- Saving of costs on DigitalOcean ([Ashish Padhy](https://github.com/shurtu-gal)): AsyncAPI was running out of credits too fast, and with [proper deployment changes](https://github.com/asyncapi/server-api/issues/317), it's not the case anymore.
+
+- GitHub Action refactored from scratch to use CLI ([Ashish Padhy](https://github.com/shurtu-gal)): Simplification is always a driver for innovation. Now official GitHub Action from AsyncAPI supports not only AsyncAPI Generator but also 1:1 all features from the CLI.
+
+- Not just any but AMAZING [cheat sheet poster](https://github.com/asyncapi/website/issues/425#issuecomment-2121214509) ([Aishat Muibudeen](https://github.com/Mayaleeeee)): It is not only available [for download](https://asyncapi.com/cheatsheet), but is also used at AsyncAPI conferences.
+
+![](/img/posts/bounty-program/362263381-acc75432-0f58-4c96-a65f-f172d34ee4ec.png)
```

Comment:
sorry, I recommended to use this photo but it is as large as 3.29 MB 😨 

please squoosh it below 1MB - https://squoosh.app/ - it doesn't have to be large picture

Reply:
Resized to 132 kB.